src/benchmark/java/com/commercetools/sync/benchmark/ProductSyncBenchmark.java

```diff
@@ -61,15 +61,15 @@
     private List<Throwable> errorCallBackExceptions;
 
     @BeforeClass
-    public static void setup() {
+    public static void setupClass() {
         deleteProductSyncTestData(CTP_TARGET_CLIENT);
         createCategoriesCustomType(OLD_CATEGORY_CUSTOM_TYPE_KEY, ENGLISH,
             OLD_CATEGORY_CUSTOM_TYPE_NAME, CTP_TARGET_CLIENT);
         productType = createProductType(PRODUCT_TYPE_RESOURCE_PATH, CTP_TARGET_CLIENT);
     }
 
     @Before
-    public void setupTest() {
+    public void setup() {
```

Comment:
what do you think about the name `setupEach`? do you think it's maybe more self expressive that means its before each test?

Reply:
I like this idea! Maybe it should be more self explaining naming what is it actually doing. And we leave the explanation of setup each or class to be explained by the annotation alone.

Reply:
Yes, The method names are for developers to understand without looking at the body of the method. The annotations are for that to make it meaningfully about the context and design. So, in this case, we are just naming it twice (btw it is nice to have the same convention if you don't think about naming these things). But probably it will take time to rename all the methods like everyone heard sth like this quote 😄 

> There are only two hard things in Computer Science: cache invalidation and naming things.
-- Phil Karlton

Reply:
Take more time yes. But definitely better :)